tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "declaration": true,
     "noImplicitOverride": true,
     "noUnusedLocals": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "types": ["vitest/globals"]
```

Comment:
you should also add `lib: 'dom'` to avoid TS compilation issues

Reply:
I'm pretty sure `dom` is included by default.

See here: https://www.typescriptlang.org/tsconfig#lib

Reply:
gotcha.
it broke down for Review Center in the past because they explicitly overrode lib with just ["ESNext"], but didn't include dom. so if default is not touched, should be fine